utils/cronjobs_osgeo_lxd/cron_grass7_relbranch_build_binaries.sh

```diff
@@ -240,18 +240,29 @@ cd $GRASSBUILDDIR
 sh ~/cronjobs/compile_addons_git.sh ~/src/grass$GMAJOR-addons/src/ \
    ~/src/$BRANCH/dist.$ARCH/ \
    ~/.grass$GMAJOR/addons \
-   ~/src/$BRANCH/bin.$ARCH/grass$VERSION
+   ~/src/$BRANCH/bin.$ARCH/grass$VERSION \
+   1
 mkdir -p $TARGETHTMLDIR/addons/
-cp ~/.grass$GMAJOR/addons/docs/html/* $TARGETHTMLDIR/addons/
+# copy indvidual addon html files into one target dir if compiled addon
+# has own dir e.g. ~/.grass8/addons/db.join/ with bin/ docs/ etc/ scripts/
```

Comment:
```suggestion
# has own dir e.g. ~/.grass7/addons/db.join/ with bin/ docs/ etc/ scripts/
```

(I guess that 7 is more appropriate in this file)

Reply:
Yes you are right, I forgot to change it when copying.